diff --git a/doxyfile.in b/doxyfile.in
index ff7bd963..f92e264a 100644
--- a/doxyfile.in
+++ b/doxyfile.in
@@ -38,7 +38,7 @@ PROJECT_NUMBER         = @VERSION@
 # If a relative path is entered, it will be relative to the location
 # where doxygen was started. If left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = 
+OUTPUT_DIRECTORY       = default/
 
 # If the CREATE_SUBDIRS tag is set to YES, then doxygen will create
 # 4096 sub-directories (in 2 levels) under the output directory of each output
@@ -574,21 +574,7 @@ WARN_LOGFILE           =
 # directories like "/usr/src/myproject". Separate the files or directories
 # with spaces.
 
-INPUT                  = @SRCDIR@/common \
-                         @SRCDIR@/posix \
-                         @SRCDIR@/macosx \
-                         @SRCDIR@/macosx/coreaudio/ \
-                         @SRCDIR@/macosx/coremidi/ \
-                         @SRCDIR@/linux \
-                         @SRCDIR@/linux/alsa \
-                         @SRCDIR@/linux/alsarawmidi \
-                         @SRCDIR@/linux/firewire \
-                         @SRCDIR@/windows \
-                         @SRCDIR@/windows/portaudio \
-                         @SRCDIR@/windows/winmme \
-                         @SRCDIR@/solaris \
-                         @SRCDIR@/solaris/oss \
-                         @SRCDIR@/common/jack/
+INPUT                  = @SRCDIR@/common/jack/
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding, which is
@@ -806,7 +792,7 @@ HTML_HEADER            =
 # each generated HTML page. If it is left blank doxygen will generate a
 # standard footer.
 
-HTML_FOOTER            =
+HTML_FOOTER            = @SRCDIR@/no_date_footer.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading
 # style sheet that is used by each HTML page. It can be used to
diff --git a/no_date_footer.html b/no_date_footer.html
new file mode 100644
index 00000000..8bf8be6b
--- /dev/null
+++ b/no_date_footer.html
@@ -0,0 +1,5 @@
+<hr size="1"><address style="text-align: right;"><small>
+Generated for $projectname by <a href="http://www.doxygen.org/index.html"><img src="doxygen.png" alt="doxygen" align="middle" border="0"></a>
+$doxygenversion</small></address>
+</body>
+</html>
