diff -rupN jack2-1.9.12.org/common/wscript jack2-1.9.12/common/wscript
--- jack2-1.9.12.org/common/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/common/wscript	2018-07-05 22:32:26.811067221 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 
 import re
diff -rupN jack2-1.9.12.org/dbus/wscript jack2-1.9.12/dbus/wscript
--- jack2-1.9.12.org/dbus/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/dbus/wscript	2018-07-05 22:32:26.812067218 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 
 import os.path
diff -rupN jack2-1.9.12.org/example-clients/jack_control jack2-1.9.12/example-clients/jack_control
--- jack2-1.9.12.org/example-clients/jack_control	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/example-clients/jack_control	2018-07-05 22:32:26.814067212 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
 
 name_base = 'org.jackaudio'
 control_interface_name = name_base + '.JackControl'
diff -rupN jack2-1.9.12.org/example-clients/wscript jack2-1.9.12/example-clients/wscript
--- jack2-1.9.12.org/example-clients/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/example-clients/wscript	2018-07-05 22:32:26.813067215 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 
 example_programs = {
diff -rupN jack2-1.9.12.org/man/wscript jack2-1.9.12/man/wscript
--- jack2-1.9.12.org/man/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/man/wscript	2018-07-05 22:32:26.813067215 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 
 import re
diff -rupN jack2-1.9.12.org/tests/wscript jack2-1.9.12/tests/wscript
--- jack2-1.9.12.org/tests/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/tests/wscript	2018-07-05 22:32:26.813067215 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 
 test_programs = {
diff -rupN jack2-1.9.12.org/waf jack2-1.9.12/waf
--- jack2-1.9.12.org/waf	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/waf	2018-07-05 22:32:26.806067234 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
 # encoding: ISO8859-1
 # Thomas Nagy, 2005-2015
 
diff -rupN jack2-1.9.12.org/wscript jack2-1.9.12/wscript
--- jack2-1.9.12.org/wscript	2017-12-13 16:19:04.000000000 -0500
+++ jack2-1.9.12/wscript	2018-07-05 22:32:26.810067223 -0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 # encoding: utf-8
 from __future__ import print_function
 
