diff -up jack2-1.9.12/doxyfile.in.nointernalapi jack2-1.9.12/doxyfile.in
--- jack2-1.9.12/doxyfile.in.nointernalapi	2017-12-26 17:43:14.387643693 +0100
+++ jack2-1.9.12/doxyfile.in	2017-12-26 17:44:34.743795861 +0100
@@ -574,22 +574,7 @@ WARN_LOGFILE           =
 # directories like "/usr/src/myproject". Separate the files or directories
 # with spaces.
 
-INPUT                  = @SRCDIR@/common \
-                         @SRCDIR@/posix \
-                         @SRCDIR@/macosx \
-                         @SRCDIR@/macosx/coreaudio/ \
-                         @SRCDIR@/macosx/coremidi/ \
-                         @SRCDIR@/linux \
-                         @SRCDIR@/linux/alsa \
-                         @SRCDIR@/linux/alsarawmidi \
-                         @SRCDIR@/linux/firewire \
-                         @SRCDIR@/linux/freebob \
-                         @SRCDIR@/windows \
-                         @SRCDIR@/windows/portaudio \
-                         @SRCDIR@/windows/winmme \
-                         @SRCDIR@/solaris \
-                         @SRCDIR@/solaris/oss \
-                         @SRCDIR@/common/jack/
+INPUT                  = @SRCDIR@/common/jack/
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding, which is
